getting_started/step_by_step/resources.rst

```diff
@@ -158,12 +158,201 @@ Freeing resources
 Resource extends from :ref:`Reference <class_Reference>`.
 As such, when a resource is no longer in use, it will automatically free
 itself. Since, in most cases, Resources are contained in Nodes, scripts
-or other resources, when a node is removed or freed, all the children
+or other resources, when a node is removed or freed, all the owned
 resources are freed too.
 
 Scripting
 ---------
 
-Like any object in Godot, not just nodes, resources can be scripted,
-too. However, there isn't generally much of an advantage, as resources
-are just data containers.
+Like any Object in Godot, users can also script Resources. Resource
+scripts grant all the same benefits as the base Resource object. They
+inherit both reference-counting memory management and automatic serialization
+between binary or text data (\*.res, \*.tres) and in-memory object
+properties.
```

Comment:
*Resource scripts grant all the same benefits as the base Resource object.
They inherit both reference-counting memory management and automatic serialization
between binary or text data (\*.res, \*.tres) and in-memory object properties.*

This is IMO very complex. We need to avoid overwhelming users with cognitive overload :)

How about:

*Resource scripts inherit all the same benefits as the base Resource object:
Automatic memory management using reference counting, automatic serialization
between binary or text data (\*.res, \*.tres) and in-memory object properties.*